### DISCUSSION TITLE
No midi sound after upgrade

### DISCUSSION THREAD

**Discussion by PietRitzen** [hydrogen-music/hydrogen#2018] [General]
Hello, 
I use hydrogen sometimes to play percussion sounds via a small program that sends a midi message to hydrogen when I push a switch. This is working fine in version 1.1.1 but after an upgrade to 1.2.0. no sound is generated. Everything looks fine. The mide settings are the same as before. The blue led flashes when a midi message is received and the log file shows the message. But no sound at all. Did I forgot something after the upgrade?
See here some log file info:
(I) MidiInput::handleMidiMessage [start of handleMidiMessage] channel: 9, val1: 40, val2: 80
(I) MidiInput::handleMidiMessage This is a NOTE ON message.
(I) MidiInput::handleMidiMessage [end of handleMidiMessage]
(I) MidiInput::handleMidiMessage [start of handleMidiMessage] channel: 9, val1: 40, val2: 80
(I) MidiInput::handleMidiMessage This is a NOTE ON message.
(I) MidiInput::handleMidiMessage [end of handleMidiMessage]
(I) MidiInput::handleMidiMessage [start of handleMidiMessage] channel: 9, val1: 40, val2: 80

Any tip to furher investigate this is welcome :-)

Best regard,
Piet

**Comment by PietRitzen** [comment]
I forgot to mention: hydrogen does produce sound. It is the midi message that does not produce sound. The rest of the program works fine.

**Comment by theGreatWhiteShark** [comment]
Hey @PietRitzen 

The version of Hydrogen you are using (1.2.0) is suffering from a bug causing this issue (https://github.com/hydrogen-music/hydrogen/issues/1751).

You could use the [latest](https://github.com/hydrogen-music/hydrogen/releases/download/1.2.3/Hydrogen-1.2.3-x86_64.AppImage) version instead or, as a workaround, disable the Preferences > MIDI System > "Discard MIDI messages after action has been triggered" option.

**Reply by PietRitzen** [reply]
Thanks you Great White Shark for your quick answer to my question. I tried the work around an sound is flowing out of the system!
